docs/quick-release-notes/main.go

```diff
@@ -0,0 +1,167 @@
+	return tagsForEachRelease
+}
+
+func getChangeLogInfo(scanner *bufio.Scanner) ([]int, []string, int, []string) {
+	// regexp for: ## 1.0.0 (2021-12-13)
+	r, err := regexp.Compile(`## \d+\.\d+\.\d+`)
+	if (err) != nil {
+		log.Fatal(err)
+	}
+	lineCount := 0
+	counterForChangeLogLength := 0
+	var versionLineNumbers []int
+	var changeLogText []string
+	var versionTitlesForEachRelease []string
+	for scanner.Scan() {
+		counterForChangeLogLength++
+		changeLogText = append(changeLogText, scanner.Text())
+		if r.MatchString(scanner.Text()) {
+			// ## 2.3.0 (2022-02-24) => 2.3.0 (2022-02-24)
+			versionNumberAndDateWithoutHashes := scanner.Text()[3:]
+			versionTitlesForEachRelease = append(versionTitlesForEachRelease, versionNumberAndDateWithoutHashes)
+			versionLineNumbers = append(versionLineNumbers, lineCount)
+		}
+		lineCount++
+	}
+	if scanner.Err() != nil {
+		fmt.Println(scanner.Err())
+	}
+	return versionLineNumbers, changeLogText, counterForChangeLogLength, versionTitlesForEachRelease
+}
+
+func makeReleaseSlice(versionLineNumbers []int, changeLogText []string, changeLogLength int) []string {
+	// Store contributors and errors from end of changelog.
+	contributorsAndErrors := ""
+	numberOfLinesForLastRelease := 5
+	for i := numberOfLinesForLastRelease + versionLineNumbers[len(versionLineNumbers)-1]; i < changeLogLength; i++ {
+		contributorsAndErrors += changeLogText[i] + "\n"
+	}
+	var releaseNotesForEachVersion []string
+	// exclude Last version (## 0.2.0) with - 1
+	versionLineNumbersLengthMinusOne := len(versionLineNumbers) - 1
+	for i, versionLineNumber := range versionLineNumbers[:] {
+		releaseBodyLines := ""
+		if i < (versionLineNumbersLengthMinusOne) {
+			for j := versionLineNumbers[i] + 1; j < versionLineNumbers[i+1]; j++ {
+				releaseBodyLines += changeLogText[j] + "\n"
+			}
+		}
+		// Handle last version (## 0.2.0)
+		if versionLineNumber == versionLineNumbers[versionLineNumbersLengthMinusOne] {
+			for j := 1; j < numberOfLinesForLastRelease; j++ {
+				releaseBodyLines += changeLogText[versionLineNumber+j] + "\n"
+			}
+		}
+		releaseNotesForEachVersion = append(releaseNotesForEachVersion, releaseBodyLines+contributorsAndErrors)
+	}
+	return releaseNotesForEachVersion
+}
+
+func sendToGitHubAPI(tagForRelease Tag, releaseNote string, versionTitle string, owner string, repo string) {
+	// https://docs.github.com/en/rest/reference/releases
+	postBody, _ := json.Marshal(map[string]string{
+		"tag_name":         tagForRelease.Name,
+		"name":             versionTitle,
+		"body":             releaseNote,
+		"target_commitish": tagForRelease.Commit.SHA,
+	})
+	responseBody := bytes.NewBuffer(postBody)
+	url := fmt.Sprintf("https://api.github.com/repos/%v/%v/releases", owner, repo)
+	req, err := http.NewRequest("POST", url, responseBody)
+	req.Header.Set("Accept", "application/vnd.github.v3+json")
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("Authorization", "token insert_github_access_token_here")
```

Comment:
Use the `flag` package to accept the token on the command line. Or accept a file containing the token.

Reply:
I will do that last otherwise its annoying while testing on my fork because my owner is different for the releases API than it is for tags API. Although I think generally using it you would have the same for both. Depends whether you want the script to be generic; I thought it was just for quick-lint-js :man_shrugging:

Reply:
> I don't want to have four flag parameters

We could have separate repos with only two flags: `-Repo LeeWannacott/quick-lint-js -TagsRepo quick-lint/quick-lint-js`

> I thought it was just for quick-lint-js 🤷‍♂️

It is. I thought command-line options would be helpful for testing.